spec/api-web-contents-spec.js

```diff
@@ -1273,4 +1273,29 @@ describe('webContents module', () => {
       expect(data).to.be.an.instanceof(Buffer).that.is.not.empty()
     })
   })
+
+  // FIXME: disable during chromium update due to crash in content::WorkerScriptFetchInitation::CreateScriptLoaderOnIO
+  xdescribe('Shared Workers', () => {
+    it('can get multiple shared workers', async () => {
+      const worker1 = new SharedWorker('../fixtures/api/shared-worker/shared-worker1.js')
+      const worker2 = new SharedWorker('../fixtures/api/shared-worker/shared-worker2.js')
+      worker1.port.start()
+      worker2.port.start()
+      const contents = webContents.getAllWebContents()
+      const vectorOfWorkers = contents[0].getAllSharedWorkers()
+
+      expect(vectorOfWorkers.length).to.equal(2)
+      expect(vectorOfWorkers[0].url).to.contain('shared-worker')
+      expect(vectorOfWorkers[1].url).to.contain('shared-worker')
+
+      contents[0].inspectSharedWorkerById(vectorOfWorkers[0].id)
```

Comment:
Please use two tests for two new methods.

Reply:
Fixed